packages/bls/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chainsafe/bls",
-  "version": "0.1.7",
```

Comment:
didnt we say were gunna update version numbers in a different PR or something?

Reply:
Might be, tbh, I've updated version just to try it out in lodestar, I can remove

Reply:
That said, if we have to release something before we can use it in monorepo, whats the differences from having multiple repos? :smile: cc @wemeetagain